app/src/main/java/com/example/dispatcher/presentation/auth/AuthViewModel.kt

```diff
@@ -0,0 +1,49 @@
+package com.example.dispatcher.presentation.favorites.viewModel
```

Comment:
this is another copy-paste that you do, this comment is repeating itself!
this is happening way to much. i don't understand how - android studio tells you that there's a problem.